sdks/java/io/google-cloud-platform/src/main/java/org/apache/beam/sdk/io/gcp/pubsublite/CloudPubsubTransforms.java

```diff
@@ -48,4 +49,21 @@ private CloudPubsubChecks() {}
               return message;
             });
   }
+
+  /**
+   * Transform messages read from Pub/Sub Lite to their equivalent Cloud Pub/Sub Message that would
+   * have been read from PubsubIO.
+   *
+   * <p>Will fail the pipeline if a message has multiple attributes per map key.
+   */
+  public static PTransform<PCollection<? extends SequencedMessage>, PCollection<PubsubMessage>>
```

Comment:
Can you write this as a composite transform? Same logic, but in the `expand` method. A bit of boilerplate that helps out UIs and metrics, etc.